database/distribution.go

```diff
@@ -197,3 +197,16 @@ func (db *Db) DeleteDelegatorRewardsAmount(delegatorAddr string, height int64) e
 
 	return nil
 }
+
+// GetDelegatorRewards returns all delegation reward data stored in db
+func (db *Db) GetDelegatorRewards() ([]*dbtypes.DelegationRewardRow, error) {
```

Comment:
Since we are using this only to check if there are some delegations, I think it would be much more efficient to have a method such as 

```go
func (db *Db) HasDelegatorRewards() bool
```

Which simply returns `true` if there are some rewards, and `false` otherwise. This can then be implemented with a more efficient query as 

```sql
SELECT COUNT(*) FROM delegation_reward
```

And then simply return `count > 0`